.template/spec/base/config/environments/production_spec.rb

```diff
@@ -9,6 +9,10 @@
     expect(subject).to contain(mailer_default_url_config)
   end
 
+  it 'allow Rails serve static file' do
```

Comment:
Should it be `allows` instead of `allow`, what do you think?

Reply:
Good catch @gutakk 👍

Reply:
Fixed in https://github.com/nimblehq/rails-templates/pull/242/commits/0d7f3767998d5694a9f0254fca1c53b3e9c5b182 @gutakk